tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -100,14 +101,16 @@
   };
 
   BrowserContextMenu.prototype.view = function() {
-    return '<form class="contextmenu" role="dialog" tabindex="-1"' +
-              ' data-type="action" ' +
-              'id="' + this.CLASS_NAME + this.instanceID + '">' +
-              '<header class="contextmenu-header"></header>' +
-              '<div class="contextmenu-list-frame">' +
-                '<menu class="contextmenu-list"></menu>' +
-              '</div>' +
-            '</form>';
+    return '<smart-dialog id="contextmenu-dialog">' +
+              '<form class="contextmenu" role="dialog" tabindex="-1"' +
```

Comment:
Since we want to use smart-dialog to replace `form`, please file a bug for this one. We still need to do it.